javascript/javascript2/week1/generator.js

```diff
@@ -1,24 +1,22 @@
-const firstNames = ["Hope", "Launa", "Katherine", "Walker", "Moses", "Tayna", "Rosia", "Yahaira", "Tommy", "Elwanda", "Clorinda", "Sheron", "Kayla", "Clementina", "Rene", "Rex", "Kathy", "Latoya", "Shirleen", "Shoshana"];
-
-const secondNames = ["Chauncey", "Houchins", "Lama", "Frasca", "Houston", "Ake", "Shankles", "Cantor", "Mizell", "Cleland", "Maltby", "Tirrell", "Cary", "Mace", "Horta", "Carlile", "Deegan", "Torrez", "Humfeld", "Helgeson"];
-
-const getRandomNumber = (max) => Math.floor(Math.random() * max);
-
-const getRandomName = () =>
-    `${firstNames[getRandomNumber(firstNames.length)]} ${secondNames[getRandomNumber(secondNames.length)]}`;
-
-const setRandomName = () => {
-    document.getElementById('random-name').innerText = getRandomName();
+const spiritAnimal = ["The fullmoon wolf", "Boring Chicken", " Lazy Lama", "Firceless Tiger", "Laughing Monkey", "Cute Cat", "A protector Dog", "Jungle big mouth Elephant", "Linked to creation Snake", "Dont judge me Rat", "Beautiful Dolphin", "Dont excit Dragon", "Explorer of the unknown Panther", "leopard", "Crezy and sharp Eagle", "Peace Dove", "Unfiend Unicorn", "Bring joy Butterfly", "King of water Starfish", "Dont look at me Tuna"];
+const buttonTag = document.getElementById('displayButton');
```

Comment:
buttonTag . I think a better name is an element not a tag. 
```suggestion
const buttonElement= document.getElementById('displayButton');
```